src/ui/json/JsonString.js

```diff
@@ -10,13 +10,7 @@ class JsonString extends Component<{
   };
 
   render() {
-    return (
-      <span>
-        {'"'}
-        {this.props.json.replace(/"/g, '\\"')}
-        {'"'}
-      </span>
-    );
+    return <span>{JSON.stringify(this.props.json)}</span>;
```

Comment:
nice

Reply:
Technically I can do the same for number, boolean and null. I didn't benchmark whether that's faster. It's the easiest way to convert a primitive type value into its string representation.